docs/docusaurus/docs/core/customize_expectations/expectation_conditions.md

```diff
@@ -39,6 +39,8 @@ Great Expectations allows you to specify conditions for validating rows using th
 
          The `condition_parser` defines the syntax of `row_condition` strings. When implementing Expectation conditions with pandas, set this argument to `"pandas"`.
 
+         Note that the Expectation with conditions will fail if the Batch being validated is from a different type of Data Source than indicated by the `condition_parser`.
```

Comment:
note for reviewer: I couldn't get this to indent properly without moving it into the tab items